feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go

```diff
@@ -605,7 +609,6 @@ func configureDUTISIS(t *testing.T, dut *ondatra.DUTDevice, aggIDs []string) {
 	for _, aggID := range aggIDs {
 		isisIntf := isis.GetOrCreateInterface(aggID)
 		isisIntf.GetOrCreateInterfaceRef().Interface = ygot.String(aggID)
-		isisIntf.GetOrCreateInterfaceRef().Subinterface = ygot.Uint32(0)
```

Comment:
why is this being removed? full ref (interface+subif) should be used here.

if an implementation doesn't support refs they should use a deviation (e.g. just below there's `if deviations.InterfaceRefConfigUnsupported(dut)` statement that can be used for them)

Reply:
That shouldn't be the case

OC definition says:
```
In contexts
        where a Layer 3 interface is to be referenced, both the
        interface and subinterface leaves must be populated, as
        Layer 3 configuration within the OpenConfig models is
        associated with a subinterface.
```

https://github.com/openconfig/public/blob/0ae6953d5d0ee8156f48088ff7a1bb37ae0ce179/release/models/interfaces/openconfig-interfaces.yang#L228-L239

Bundles can also have subinterfaces, and you can see it is being configured on https://github.com/openconfig/featureprofiles/blob/99f8453fdaa2c47f9bffc41828cd16e5edece029/feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go#L452-L467 in this test.

The issue is that some vendors don't use proper subinterfaces and associate layer 3 information with the physical interface when you configure "subinterface 0", but it's not an OC-compliant way.

cc @dplore